### PR TITLE
Wish Presence

### PR DESCRIPTION
**Pull request by MauriceX24**
jKTX3v0o

### CODE REVIEW

File: Wish/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "name": "MauriceX",
+        "id": "268768664411439115"
+    },
+    "service": "Wish",
+    "description": {
+        "en": "Buy many products with 80% discount.",
+        "de": "Kaufe viele Produkte mit 80% Rabatt."
+    },
+    "url": [
+        "wish.com",
```
Comment (frozaum) [inline Wish/dist/metadata.json]: remove this since they only use www.wish.com

File: Wish/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		"www.wish.com"
+    ],
+    "version": "1.0",
+    "logo": "https://i.imgur.com/XsZ2IOn.jpg",
+    "thumbnail": "https://i.imgur.com/R7AkWiP.png",
+    "color": "#48c0eb",
+    "tags": [
+        "Shopping",
+		"Shop"
```
Comment (frozaum) [inline Wish/dist/metadata.json]: unnecessary spacing

Comment (MauriceX24) [reply Wish/dist/metadata.json]: Done.

File: Wish/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    "author": {
+        "name": "MauriceX",
+        "id": "268768664411439115"
+    },
+    "service": "Wish",
+    "description": {
+        "en": "Buy many products with 80% discount.",
+        "de": "Kaufe viele Produkte mit 80% Rabatt."
+    },
+    "url": [
```
Comment (Bas950) [inline Wish/dist/metadata.json]: just make it: "url": "www.wish.com",

Comment (MauriceX24) [reply Wish/dist/metadata.json]: Done 👍

**Pull request merged**